packages/paste-core/components/sidebar/__tests__/index.spec.tsx

```diff
@@ -98,6 +116,27 @@ describe('Sidebar', () => {
     });
   });
 
+  /**
+   * SIDEBAR COLLAPSE BUTTON
```

Comment:
suggestion: test that the correct icon and label render depending on `collapsed`